packages/discovery-provider/src/api/v1/helpers.py

```diff
@@ -874,11 +874,12 @@ def add_auth_headers_to_parser(parser):
 
 
 def success_response(entity):
-    return api_helpers.success_response(entity, 200, False)
+    return api_helpers.success_response(entity, status=200, to_json=False)
 
 
 def error_response(error):
-    return api_helpers.error_response(error)
+    # This is not really a success, but we care about getting the error data back in the same shape
```

Comment:
what's the motivation for this? why not just return a regular `error_response`?

Reply:
it stinks. the response marshalers for all v1 expects a "data: x" object and error doesn't work for that because it has success/error fields.